griptape/tools/image_gen/tool.py

```diff
@@ -0,0 +1,46 @@
+import os
+import openai
+from griptape.artifacts import BlobArtifact, ListArtifact, TextArtifact
+from griptape.tools import BaseTool
+from griptape.utils.decorators import activity
+from schema import Schema, Optional
+import requests
+
+
+class ImageGenerator(BaseTool):
+    api_key = os.environ.get("OPENAI_API_KEY")
```

Comment:
We should define a field here with a sensible default:

```suggestion
    api_key: Optional[str] = field(
        default=Factory(lambda: os.environ.get("OPENAI_API_KEY")), kw_only=True
    )
```

Reply:
Done!